spring-data-vmware-gemfire/build.gradle

```diff
@@ -12,7 +12,7 @@ plugins {
 }
 
 project.ext.set("pomProjectLongName", "Spring Data VMware GemFire")
-project.ext.set("pomProjectArtifactName", "spring-data-3.1-gemfire-${getGemFireBaseVersion()}")
+project.ext.set("pomProjectArtifactName", "spring-data-${getSpringDataBaseVersion()}-gemfire-${getGemFireBaseVersion()}")
```

Comment:
This change is not consistent with that of the previous submissions. Please revert or do the same for all the other projects

Reply:
Fixed